tools/PI/DevHome.PI/BarWindow.cs

```diff
@@ -141,4 +129,24 @@ public void RotateBar()
             _horizontalWindow.Show();
         }
     }
+
+    public void UpdateBarWindowPosition(PointInt32 position)
```

Comment:
This setter is public.  Can `_viewModes.WindowPosition` be made a public property with a public set?

Reply:
No, we want to set WindowPosition only from BarWindow without exposing the viewmodel.